### PR TITLE
#457 - add back kpt deploy instructions for core-landing-zone - ready

### PR DESCRIPTION
**Pull request by fmichaelobrien**
gatekeeper optional - just instructions for core-landing-zone

Test results of all script additions in #457

### CODE REVIEW

**Comment by fmichaelobrien** [comment]
fixing linting

```
2023-08-15 01:31:48 [INFO]   Successfully found:[GITHUB_REPO], value:[pubsec-declarative-toolkit]
2023-08-15 01:31:48 [ERROR]   Failed to get [GITHUB_TOKEN]!
2023-08-15 01:31:48 [ERROR]   []
2023-08-15 01:31:48 [ERROR]   Please set a [GITHUB_TOKEN] from the main workflow environment to take advantage of multiple status reports!

2023-08-15 01:31:53 [INFO]   File:[/github/workspace/docs/landing-zone-v2/README.md]
2023-08-15 01:31:53 [ERROR]   Found errors in [markdownlint] linter!
2023-08-15 01:31:53 [ERROR]   Error code: 1. Command output:
------
/github/workspace/docs/landing-zone-v2/README.md:3[81](https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/actions/runs/5862548542/job/15894515244?pr=458#step:6:82) MD046/code-block-style Code block style [Expected: fenced; Actual: indented]
/github/workspace/docs/landing-zone-v2/README.md:3[87](https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/actions/runs/5862548542/job/15894515244?pr=458#step:6:88) MD046/code-block-style Code block style [Expected: fenced; Actual: indented]
/github/workspace/docs/landing-zone-v2/README.md:3[93](https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/actions/runs/5862548542/job/15894515244?pr=458#step:6:94) MD046/code-block-style Code block style [Expected: fenced; Actual: indented]
/github/workspace/docs/landing-zone-v2/README.md:399 MD046/code-block-style Code block style [Expected: fenced; Actual: indented]
/github/workspace/docs/landing-zone-v2/README.md:405 MD046/code-block-style Code block style [Expected: fenced; Actual: indented]
------
```

**Comment by obriensystems** [comment]
I'll fix the linting later today after meets - linting takes longer that the actual PR sometimes

**Comment by fmichaelobrien** [comment]
ready for review/merge

File: docs/landing-zone-v2/README.md [hunk 1]
```diff
@@ -366,6 +368,53 @@ cd pbmm-landingzone
     Review and customize all packages' `setters.yaml` with the unique configuration of your landing zone.  
     For example "core-landing-zone" will have the same [setters.yaml](https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/main/solutions/landing-zone-v2/setters.yaml) as in the repo in the root of the pkg directory.
 
+### <a name='deploy-the-infrastructure-using-kpt'></a>2b. Deploy the infrastructure using KPT
+This section is an alternative to GitOps in the following section 3.
+
+#### <a name='gatekeeper-policies-kpt'></a>gatekeeper-policies
+Optional
```
Comment (cartyc) [inline docs/landing-zone-v2/README.md:375]: Could we add something along the lines of strongly recommended or required for Prod?

**Review by cartyc — CHANGES_REQUESTED** [review]
Looks good aside from the one nit

**Review by cartyc — APPROVED** [review]

**Comment by fmichaelobrien** [comment]
merge this one before https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/pull/476

**Review by alaincormier-ssc — APPROVED** [review]
lgtm

**Comment by fmichaelobrien** [comment]
Merging and will rebase with https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/pull/476

**Pull request merged**